sdks/python/apache_beam/io/filesystems.py

```diff
@@ -17,24 +17,47 @@
 
 """FileSystems interface class for accessing the correct filesystem"""
 
+import re
+
 from apache_beam.io.filesystem import BeamIOError
 from apache_beam.io.filesystem import CompressionTypes
-from apache_beam.io.filesystems_util import get_filesystem
+from apache_beam.io.filesystem import FileSystem
+
+# pylint: disable=wrong-import-position, unused-import
+from apache_beam.io.localfilesystem import LocalFileSystem
```

Comment:
Not sure I understand that completely but we want to know about the FS without the user process actually instantiating anything so I think do need the imports